tests/models/test_model_talk.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+
+from giskard import scan
+
+
+@pytest.mark.parametrize(
+    "dataset_name,model_name",
+    [
+        ("titanic_dataset", "titanic_model"),
+        ("hotel_text_data", "hotel_text_model"),
+    ],
+)
+def test_talk(dataset_name, model_name, request):
+    _default_question = "What can you do?"
+
+    # Fetch dataset and model.
+    dataset = request.getfixturevalue(dataset_name)
+    model = request.getfixturevalue(model_name)
+
+    # Get the scan report.
+    scan_report = scan(model, dataset)
+
+    # [1] Check, if an exception raised, when necessary parameters are omitted.
+    with pytest.raises(TypeError):
+        model.talk()
```

Comment:
better to intercept the message and check it against the code. See https://github.com/Giskard-AI/giskard/blob/c78b65b5c74c6e7d93f3f7e627c1bda31ef6c913/tests/test_dataset.py#L104-L109